python/paddle/fluid/layers/nn.py

```diff
@@ -6157,6 +6158,46 @@ def relu(x, name=None):
     return out
 
 
+def selu(x, scale=1.0507, alpha=1.6732, name=None):
```

Comment:
Set scale/alpha to None, so we can use C++ attribute

Reply:
Done